### PR TITLE
Make rows selectable in 2.6

### PR DESCRIPTION
**Pull request by moretaste**
Fixed issue https://github.com/symphonists/tracker/issues/27

### CODE REVIEW

File: content/content.index.php [hunk 1]
```diff
@@ -108,7 +108,8 @@ public function view()
         // Assemble the table
         $table = Widget::Table(
             Widget::TableHead($thead), null,
-            Widget::TableBody($tbody), null
+            Widget::TableBody($tbody), 'selectable', null,
+            array('role' => 'directory', 'aria-labelledby' => 'symphony-subheading', 'data-interactive' => 'data-interactive')
         );
         $table->setAttribute('class','selectable');
```
Comment (nilshoerrmann) [inline content/content.index.php]: I think you can delete this line because you already set the class in the widget call.

**Comment by moretaste** [comment]
Removed the setAttribute line

**Comment by nilshoerrmann** [comment]
If you mind updating the meta file as well, I can merge this pull request for you.

**Comment by moretaste** [comment]
Changed the https://github.com/moretaste/tracker/blob/master/extension.meta.xml

**Comment by nilshoerrmann** [comment]
Thanks!

**Pull request merged**